### DISCUSSION TITLE
error: index.js: [BABEL]: Cannot find module '../../modules/es6.object.to-string'

### DISCUSSION THREAD

**Discussion by AbdulSamad-khan** [babel/babel#13540] [Q&A] [answered]
error: index.js: [BABEL]: Cannot find module '../../modules/es6.object.to-string'
Require stack:
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-runtime\node_modules\core-js\library\fn\symbol\index.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-runtime\core-js\symbol.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-runtime\helpers\typeof.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-runtime\helpers\possibleConstructorReturn.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-core\lib\transformation\file\index.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-core\lib\api\node.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-core\index.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-preset-react-native\transforms\transform-regenerator-runtime-insertion.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-preset-react-native\configs\main.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-preset-react-native\index.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\@babel\core\lib\config\files\module-types.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\@babel\core\lib\config\files\configuration.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\@babel\core\lib\config\files\index.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\@babel\core\lib\index.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\metro-transform-worker\src\index.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\metro\src\DeltaBundler\Worker.js
- C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\metro\node_modules\jest-worker\build\workers\processChild.js (While processing: C:\Users\Mudasir\OneDrive\Desktop\Android-Vendi-updated-Gradle\node_modules\babel-preset-react-native\index.js)

Getting this error on the development server metro while working with react native cli. The project was initially created in early 2019 but I have fixed the other version dependencies and my build is also successful but only problem is development server is getting above error...

**Comment by nicolo-ribaudo** [comment, marked as answer]
What is the `babel-runtime` version?

Also, the problem might be caused by mixing Babel 6 and Babel 7 (they are usually not compatible with each other): in your stack trace you have both `babel-core` (Babel 6) and `@babel/core` (Babel 7).

**Reply by AbdulSamad-khan** [reply]
Thank you!
